.mvn/jvm.config

```diff
@@ -2,3 +2,5 @@
 -Dmaven.wagon.http.retryHandler.requestSentEnabled=true
 -Dmaven.wagon.http.retryHandler.count=10
 -Dkotlin.environment.keepalive=true
+-Xmx8g
```

Comment:
@gsmet this would help a lot as people would no longer need to read the instructions prior to importing the project.

But do we have enough physical RAM on CI?

Reply:
I am pretty sure that RAM capacity of CI is 8GB

Reply:
cool, thanks @geoand

Reply:
You mean 7 GB? :-) https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources

Reply:
ah, thanks @Ladicek . I'll change it to 5 - should be enough.

Reply:
BTW doesn't CI override it? I'll change it to 5 just to be sure but I wonder for how long it will be enough :)

Reply:
> You mean 7 GB? :-) https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources

Ah, good point!

Reply:
I must say it's a bit sad that just for one IDE (albeit the most popular one as it seems) we now have to set almost three times as much memory than before.
I guess folks with constrainted memory (and who are not using IDEA) can just lower it via `MAVEN_OPTS`.

But setting _some_ common value via `jvm.config` was long overdue. 👍

Reply:
yea although to be fair it's unclear how much it's really needing.  From what I heard, it's working OK for most people provided they set 2G but some mentioned having to set more; my guess is that 2G is good enough but there's possibly other problems confusing such observations...

So that's why I went with a min and a max heap setting; max to 5G but I hope it will never actually reach to using them all, probably not even half.